fixtures/tracks/animal/SETUP.md

```diff
@@ -0,0 +1 @@
+This is the content of the setup.md file.
```

Comment:
No I do not think it did. I ran the tests locally. Since this is a new file in animal track, not other tests should use it.